packages/design-system/package.json

```diff
@@ -97,8 +97,8 @@
     "typescript": "^2.9.1"
   },
   "peerDependencies": {
-    "react": "16",
-    "react-dom": "16"
+    "react": "16 || 17",
```

Comment:
can we enforce at least version [16.13.1](https://github.com/facebook/react/blob/master/CHANGELOG.md#16131-march-19-2020) here?:) I would like to have react version with hooks and some improvements